# Configuration file for the microphone
camera:
  default_param:
    input_device_index: 0
    show: True
    video_format: "rgb8"
    fps: 30
